pygmt/tests/test_datasets_samples.py

```diff
@@ -109,9 +108,7 @@ def test_hotspots():
     """
     Check that the @hotspots.txt dataset loads without errors.
     """
-    with pytest.warns(expected_warning=FutureWarning) as record:
-        data = load_hotspots()
-        assert len(record) == 1
+    data = load_sample_data(name="hotspots")
     assert data.shape == (55, 4)
     assert list(data.columns) == [
         "longitude",
```

Comment:
The following is primarily meant for understanding. Why we are here not verifying the minimum and maximum of the "longitude" and "latitude" columns as for some of the other sample datasets?

Reply:
Good point. I think we should do the check.

Reply:
Also check min/max of symbol_size.